content/en/docs/concepts/storage/volumes.md

```diff
@@ -919,388 +654,56 @@ A container using a projected volume source as a [`subPath`](#using-subpath) vol
 receive updates for those volume sources.
 {{< /note >}}
 
-### quobyte
+### secret 
 
-A `quobyte` volume allows an existing [Quobyte](https://www.quobyte.com) volume to
-be mounted into your Pod.
+A `secret` volume is used to pass sensitive information, such as passwords, to
+Pods. You can store secrets in the Kubernetes API and mount them as files for
+use by pods without coupling to Kubernetes directly. `secret` volumes are
+backed by tmpfs (a RAM-backed filesystem) so they are never written to
+non-volatile storage.
 
 {{< note >}}
-You must have your own Quobyte setup and running with the volumes
-created before you can use it.
+A container using a Secret as a [`subPath`](#using-subpath) volume mount doesn't receive Secret updates.
 {{< /note >}}
 
-Quobyte supports the {{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}}.
-CSI is the recommended plugin to use Quobyte volumes inside Kubernetes. Quobyte's
-GitHub project has [instructions](https://github.com/quobyte/quobyte-csi#quobyte-csi) for deploying Quobyte using CSI, along with examples.
+For more details, see [Configuring Secrets](/docs/concepts/configuration/secret/).
 
-### rbd
 
-An `rbd` volume allows a
-[Rados Block Device](https://docs.ceph.com/en/latest/rbd/) (RBD) volume to mount into your
-Pod. Unlike `emptyDir`, which is erased when a pod is removed, the contents of
-an `rbd` volume are preserved and the volume is unmounted. This
-means that a RBD volume can be pre-populated with data, and that data can
-be shared between pods.
+## Storage plugin interfaces
 
-{{< note >}}
-You must have a Ceph installation running before you can use RBD.
-{{< /note >}}
+The out-of-tree volume plugins include the Container Storage Interface (CSI)
+and FlexVolume. They enable storage vendors to create custom storage plugins. The main Kubernetes binary implements the CSI, and communicates with any external plugins. This keeps the main binary efficient, and allows users to choose their own storage plugins. 
```

Comment:
There's many Kubernetes binaries. Also they implement both CSI and FlexVolume interfaces.

Reply:
Yep, it's wrong to write “the main Kubernetes binary”. It's OK to mention having code (not that 3.8 GiB is _that_ small) in-tree and easier ways for vendors to work at their own pace.